wolfcrypt/src/ecc.c

```diff
@@ -88,10 +88,14 @@ ECC Curve Sizes:
 #endif
 
 /* Make sure ASN is enabled for ECC sign/verify */
-#if (defined(HAVE_ECC_SIGN) || defined(HAVE_ECC_VERIFY)) && defined(NO_ASN)
+#if (defined(HAVE_ECC_SIGN) && defined(NO_ASN))
     #error ASN must be enabled for ECC sign/verify
 #endif
 
+#ifdef NO_ASN
+#   define ECC_SECP256R1_OID  526
```

Comment:
Its probably better to expose these in asn_public.h even with NO_ASN is defined. I suspect we'll need other OID's.

Reply:
Moved the whole enum Ecc_Sum to asn_public.h